lib/actions/action.js

```diff
@@ -47,6 +47,18 @@ class Action extends EventAware {
     }
     this.log.info(JSON.stringify(usageLog))
   }
+
+  /**
+   * Returns the actionable issues and/or pulls if it's for a schedule event. Otherwise return the current issue or pull that is for the current event.
+   */
+  getActionables (context, results) {
+    let scheduleResults = results && results.validationSuites && results.validationSuites.length && results.validationSuites[0].schedule
```

Comment:
We could instead check for `context.event === 'schedule'` and then assign this, as it sometimes fails for other events.

Reply:
since this piece of code is verbatim refactoring, does the issue you mentioned currently exists in our production?

Reply:
yup, although this works as expected, it's simply more explicit about what is expected to have these attributes. That way we can debug errors more easily. Currently in production it only fails if someone explicitly adds an event listener which we don't support for a particular action.

Reply:
I prefer to be explicit as well. The suggestion to use `context.event` is a good one.